<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UMLOperationDialog</class>
 <widget class="QWidget" name="UMLOperationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="DocumentationWidget" name="documentationWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="VisibilityEnumWidget" name="visibilityWidget" native="true"/>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>General Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="UMLDatatypeWidget" name="dataTypeWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="queryConstCB">
        <property name="text">
         <string>Query (&quot;const&quot;)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="UMLStereotypeWidget" name="stereotypeWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="abstractCB">
          <property name="text">
           <string>Abstract Operation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="classifierScopeCB">
          <property name="text">
           <string>Classifier scope (&quot;static&quot;)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" rowspan="2">
       <widget class="QListWidget" name="parametersListLW">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="newParameterPB">
          <property name="text">
           <string>New Parameter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePB">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="propertiesPB">
          <property name="text">
           <string>Properties</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="downTB">
        <property name="text">
         <string/>
        </property>
        <property name="arrowType">
         <enum>Qt::DownArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="upTB">
        <property name="text">
         <string/>
        </property>
        <property name="arrowType">
         <enum>Qt::UpArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UMLDatatypeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/umldatatypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UMLStereotypeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/umlstereotypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VisibilityEnumWidget</class>
   <extends>QWidget</extends>
   <header>widgets/visibilityenumwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DocumentationWidget</class>
   <extends>QWidget</extends>
   <header>widgets/documentationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
